src/clarity-angular/tabs/vtabs-nav-group.ts

```diff
@@ -0,0 +1,25 @@
+
+import {Component} from "@angular/core";
+
+/**
+ * @class VTabsNavGroup
+ *
+ * @description
+ * A child component of VTabs that is a container for a group of nav links.
+ * An optional label can be passed in to give the group a title.
+ * clr-vtabs-nav-links are projected to the VTabsNavLink class.
+ *
+ * Style is set in the template, not dynamically.
+ */
+@Component({
+    selector: "clr-vtabs-nav-group",
+    template: `
+        <div class="nav-group">
+            <ng-content select="label"></ng-content>
```

Comment:
The new wizard queries some elements by class, tag-name, or even attribute. And that was @youdz idea... so there is clearly a disconnect.

The one thing I would say is that the classnames we query by are generally internal and not used by the CSS for styling. That's what I would try to follow (such that if we decide to change the CSS naming convention we won't break the component).

The only thing I would ping here is that label is a bit too general to the implementation. I actually don't have a huge problem with this though.